src/build.d

```diff
@@ -10,11 +10,6 @@ detab, tolf, install targets - require the D Language Tools (detab.exe, tolf.exe
 
 zip target - requires Info-ZIP or equivalent (zip32.exe)
   http://www.info-zip.org/Zip.html#Downloads
-
-TODO:
-- add all posix.mak Makefile targets
```

Comment:
Those comments haven't been updated since 2018-05-25. As far as I can see, the only major thing in `posix.mak` remaining is the bootstrapping support, which we obviously can't move to `build.d`.